cmd/template/cluster/flag.go

```diff
@@ -388,6 +427,22 @@ func (f *flag) Validate() error {
 			if len(f.ControlPlaneAZ) > 1 {
 				return microerror.Maskf(invalidFlagError, "--%s supports one availability zone only", flagControlPlaneAZ)
 			}
+		case key.ProviderVSphere:
+			if f.VSphere.ServiceLoadBalancerCIDR == "" {
+				return microerror.Maskf(invalidFlagError, "CIDR range from which the public IPs for Services of type LoadBalancer are taken (required) (--%s)", flagVSphereServiceLoadBalancerCIDR)
+			}
+			if !validateCIDR(f.VSphere.ServiceLoadBalancerCIDR) {
+				return microerror.Maskf(invalidFlagError, "--%s must be a valid CIDR", flagVSphereServiceLoadBalancerCIDR)
+			}
+			if f.KubernetesVersion == defaultKubernetesVersion {
```

Comment:
What if I want to use `v1.20.9` for CAPV? :D I guess there is a way to check whether the input is empty or not. Can we rely on that data?

Reply:
right, not very likely at the moment, but it can be a risk in the future if it's something not that old. Check the https://github.com/giantswarm/kubectl-gs/pull/1129/commits/92d53796b3cd1e9f299b95831c7d841d55bb4e1e